lib/templates/structure/loop.php

```diff
@@ -14,90 +16,88 @@
  */
 do_action( 'beans_before_loop' );
 
-	if ( have_posts() && ! is_404() ) :
+if ( have_posts() && ! is_404() ) :
 
-		/**
+	/**
```

Comment:
The rest of this comment needs to align left too to align with its opening `/**`.